### ISSUE TITLE
crash 

### ISSUE THREAD

**Issue by skyindfo** [alexiscn/MTTransitions#19]
'Invalid parameter not satisfying: [kernel.alphaTypeHandlingRule _canHandleAlphaTypesInImages:images]'

**Comment by alexiscn** [comment]
Can you provide a demo project ?

**Comment by tikamsingh** [comment]
Yes, the older version is working fine, but in the new version I am also getting crash In this place, look at the screenshot for more detail.
![Screenshot 2024-08-27 at 3 30 41 PM](https://github.com/user-attachments/assets/01c301a8-babf-425c-9db5-8959438c9536)
